pkg/controller/cloudlet_api.go

```diff
@@ -2857,3 +2857,66 @@ func redactAccessVars(accessVars map[string]string) map[string]string {
 	}
 	return vars
 }
+
+func canRunRefreshCerts(cloudlet *edgeproto.Cloudlet, features *edgeproto.PlatformFeatures) bool {
```

Comment:
Do you need `feature`  argument, or you just wanted to make this generic?

Reply:
I just wanted to make it generic, as some platforms may not need to cert refresh.